tools/Automatic Transcription of Dutch Speech Recordings (Wav file).json

```diff
@@ -24,7 +24,7 @@
         "topic": "OH"
     },
     "mimetypes": [
-        "audio/vnd.wave"
+        "audio/wav"
```

Comment:
You should probably keep the old version, see https://github.com/clarin-eric/switchboard-tool-registry/pull/79. See also this: https://projects.softwareplumbers.com/document-management/tika/commit/bada130efe32560a5b3c26472029f3bd536040b8.